rad/src/script.rs

```diff
@@ -273,14 +276,14 @@ fn test_execute_radon_script() {
             Some(vec![Value::Text(String::from("main"))]),
         ),
         (
-            RadonOpCodes::MapGetFloat,
+            RadonOpCodes::MapGetInteger,
```

Comment:
This is exactly the issue we are trying to solve, now if the temperature changes from -4ºC to -4.5ºC, this data request breaks. So we should find a way to accept both -4 and -4.5 as valid floats, instead of modifying this test to work on integers.

Reply:
If you want that behavior, you can just do `MapGetFloat` instead of `MapGetInteger` and it works as well, I guess. 

Do you prefer if I leave the script in the test just as it was? Maybe the change I made is too reckless and a bad example because it can break easily.

Reply:
I've updated the test to illustrate my point :wink:

Reply:
I expected the test to not change to make sure we don't break anything. I guess this test was not using `MapGetInteger` because that didn't work, so we also need to add a new test using `MapGetInteger`.

Reply:
Let me see what I can do.